client/package.json

```diff
@@ -9,57 +9,54 @@
     "lint": "ng lint",
     "e2e": "ng e2e",
     "cy:open": "cypress open",
-    "docker-build": "npm run build -- --configuration production"
+    "docker-build": "npm run build"
```

Comment:
This still needs to be removed.

Reply:
I thought I made all the associated changes so we could remove these flags. Am I missing something?